src/test/arbitrum/LdoArbPayloadTest.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import {AaveGovernanceV2, AaveV3Arbitrum} from 'aave-address-book/AaveAddressBook.sol';
+import {ProtocolV3TestBase} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {ChangeLdoEmissionAdminPayload} from '../../contracts/arbitrum/LdoArbPayload.sol';
+import {TestWithExecutor} from 'aave-helpers/GovHelpers.sol';
+
+
+interface IEmissionManager {
+   /**
+   * @dev Returns the admin of the given reward emission
+   * @param reward The address of the reward token
+   * @return The address of the emission admin
+   */
+  function getEmissionAdmin(address reward) external view returns (address);
+
+  /**
+   * @dev Sets the end date for the distribution
+   * @param asset The asset to incentivize
+   * @param reward The reward token that incentives the asset
+   * @param newDistributionEnd The end date of the incentivization, in unix time format
+   **/
+  function setDistributionEnd(
+    address asset,
+    address reward,
+    uint32 newDistributionEnd
+  ) external;
+}
+
+contract LdoArbPayloadTest is
+  ProtocolV3TestBase,
+  TestWithExecutor
+{
+  ChangeLdoEmissionAdminPayload public proposalPayload;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('arbitrum'));
+    _selectPayloadExecutor(AaveV3Arbitrum.ACL_ADMIN);
+  }
+
+  function testLdoEmissionAdminArb() public {
+    proposalPayload = new ChangeLdoEmissionAdminPayload();
+    
```

Comment:
Test who the admin was before.

Reply:
can you elaborate why it's beneficial to test the current admin address? I usually treat the protocol as perfect blackbox that does not need to be tested. Only my interactions with it

Reply:
I would just for sanity check and completeness sake